### ISSUE TITLE
review of tutorials/compilation.rst

### ISSUE THREAD

**Issue by dogoepp** [resibots/limbo#125]
This is for easier discussion of pull request #115.

# `tutorials/compilation.rst`

## around line 31:
```
* `NLOpt <http://ab-initio.mit.edu/wiki/index.php/NLopt>`_ with C++ binding: ::

    ./configure -with-cxx --enable-shared --without-python --without-matlab --without-octave
    make install
```

Are you sure for the single dash in -with-cxx ?
## around line 102
 Is the list of executed tests still the same ? I saw discussions that seemed to mention new tests.

**Comment by costashatz** [comment]
> Are you sure for the single dash in -with-cxx ?

It's a double dash: `--with-cxx`

> Is the list of executed tests still the same ? I saw discussions that seemed to mention new tests.

No. There are no tests by default now.

I will updated them both..

**Comment by costashatz** [comment]
Fixed in 21ac771. **Closing...**
